src/gradleinttest.sh

```diff
@@ -37,13 +37,10 @@ gradleinttest () {
     echo "Running ${test_to_run} test with data project env var unset: ${google_data_project}"
   fi
 
-  if [[ -n "${google_project}" ]] && [ -f jade-dev-account.json ] && [ -f jade-dev-account.pem ] && [[ "${test_to_run}" != "" ]]; then
+  if [[ -n "${google_project}" ]] && [ -f ${GOOGLE_APPLICATION_CREDENTIALS} ] && [[ "${test_to_run}" != "" ]]; then
     export PGHOST=$(ip route show default | awk '/default/ {print $3}')
     export DB_DATAREPO_URI="jdbc:postgresql://${PGHOST}:5432/datarepo"
     export DB_STAIRWAY_URI="jdbc:postgresql://${PGHOST}:5432/stairway"
-    export GOOGLE_APPLICATION_CREDENTIALS=jade-dev-account.json
-    export IT_JADE_PEM_FILE_NAME=jade-dev-account.pem
-    export GOOGLE_SA_CERT=jade-dev-account.pem
```

Comment:
Just so I'm following -- you removed these env variable definitions because they're already defined somewhere else? Where could I see that?

Reply:
I made a few changes on the data-repo side of things to accommodate this:
* [Added env variables](https://github.com/DataBiosphere/jade-data-repo/pull/575/files#diff-0bdb3bdd6301a2f77d38ec352e8be069R4) on the pr build action
* [Now pointing the it.jadePemFileName to GOOGLE_SA_CERT](https://github.com/DataBiosphere/jade-data-repo/pull/575/files#diff-3ea0c674627e739883a70b46fd606762R3)

Reply:
Got it, thanks for the pointers!